app/domain/util/open_ssl/x509/quick_csr.rb

```diff
@@ -0,0 +1,91 @@
+# Currently this is useful mainly for tests, and allows you to quickly create CSRs with alt name extensions using a simple declarative interface
```

Comment:
If this is only being used in tests, should we move it to `spec/support`? Maybe not worth it if it's about to be split into a gem.

Reply:
Yes plan for this is to gem-ify.  Maybe this Friday?